dramatiq/middleware/shutdown.py

```diff
@@ -132,34 +134,41 @@ def shutdown(self):
             raise_thread_exception(thread_id, Shutdown)
 
 
-_GeventShutdownManager: Optional[Type[_ShutdownManager]] = None
```

Comment:
rather than adding the `if TYPE_CHECKING` block,
can you try changing just this first line to 
`_GeventShutdownManager: Type[_ShutdownManager]`

so rather than `_GeventShutdownManager` being either `None`/the actual class (two different types),
it is undefined / the actual class (which is one type).

Reply:
Hm. Looking at the code I _assume_ that the intention of
```python
_GeventShutdownManager = None
if is_gevent_active():   
    ...
    _GeventShutdownManager = __GeventShutdownManager
```
is to make the name available for the middleware initialization. But both uses of `_GeventShutdownManager` are guarded by `is_gevent_active()`. So, I removed the module variable `_GeventShutdownManager` and renamed the class `__GeventShutdownManager` to `_GeventShutdownManager` and the code still checks and tests just fine on my local machine:
```
=================================== 175 passed, 131 skipped, 1 warning in 241.83s (0:04:01) ===================================
```
Let me push the changes, and please do take a careful look as well.

Reply:
hmmm I can't see a reason for `__GeventShutdownManager`.
It is clearly a private name, so I'm happy to get rid of it.

Overall I think this is a good simplification.